docs/de/reference/status-snippet.md

```
@@ -1,19 +1,16 @@
 ---
 language: de
 translation_status: complete
-doc_version: '2025.11'
-last_updated: '2025-11-23'
+doc_version: '2025.12'
+last_updated: '2025-12-10'
 ---
 
-> **Systemstatus (2025-11-23) — Production Ready v0.61.3**
+> **Systemstatus (n/a) — **
```

markdownlint-cli2 (0.18.1)

Spaces inside emphasis markers

(MD037, no-space-in-emphasis)